src/core_plugins/kibana/server/lib/import/import_dashboards.js

```diff
@@ -0,0 +1,33 @@
+import { flatten } from 'lodash';
+import { SavedObjectsClient } from '../../../../../server/saved_objects';
+
+const acceptableTypes = [
```

Comment:
I'm going to remove this since this is left over from when I was working directly with the Kibana index. The SavedObject API should be doing some kind of validation on it's end (and if it's not then an issue needs to be filed) @tylersmalley